doc/api/events.markdown

```diff
@@ -137,10 +137,15 @@ Execute each of the listeners in order with the supplied arguments.
 Returns `true` if event had listeners, `false` otherwise.
 
 
+### emitter.listenerCount(type)
+
+Returns the number of listeners listening to the `type` of event.
+
 ### Class Method: EventEmitter.listenerCount(emitter, event)
 
 Return the number of listeners for a given event.
 
+_Note: This is deprecated. Use <EventEmitter object>.listenerCount instead._
```

Comment:
Don't use angle brackets. The combination of spacing and italics is causing the words to be removed.